src/App.jsx

```diff
@@ -1,60 +1,178 @@
-import "./App.css"
+import { useState } from "react";
+import "./App.css";
 
 function App() {
+  const [panelOne, setPanelOne] = useState("0");
+  const [panelTwo, setPanelTwo] = useState("0");
+  const [panelMath, setPanelMath] = useState("+");
+  const [answer, setAnswer] = useState(0);
+
+  const addToPanelOne = (event) => {
```

Comment:
there's no need to use `event`, since you could simply pass the "number" to add directly as a parameter (see my comment later down)